spdk.spec

```diff
@@ -36,8 +36,7 @@ License: BSD
 ExclusiveArch: x86_64
 
 BuildRequires: gcc gcc-c++ make
-# dpdk 18.11 is in "extras" so pin it to our version
-BuildRequires: dpdk-devel = 19.02
+BuildRequires: dpdk-devel = 19.11
```

Comment:
So this means you are going to have to update [daos-stack/dpdk](https://github.com/daos-stack/dpdk) as well then and before you can update SPDK.

You can then add a:

```
PR_REPOS := dpdk@PR-<PR-number>
```

after line 3 or so of the `Makefile` to make that dpdk available to this spdk build.